app/src/components/v-field-list/v-field-list.vue

```diff
@@ -37,16 +48,18 @@ interface Props {
 	disabledFields?: string[];
 	includeFunctions?: boolean;
 	includeRelations?: boolean;
+	canSelectAll?: boolean;
```

Comment:
Lets rename this to `allowSelectAll` to be consistent with the `v-select`

Reply:
Fixed 👍